packages/ckeditor5-math/src/mathui.ts

```diff
@@ -141,6 +166,21 @@ export default class MathUI extends Plugin {
 			this.formView.mathInputView.fieldView.element?.select();
 		}
 
+		// Allow the textarea to be resizable and observe its resize events.
+		const textarea = this.formView?.mathInputView.fieldView.element;
+		if (textarea) {
+			textarea.style.resize = 'both';
+			textarea.style.height = '100px';
+			textarea.style.width = '100%';
+			this._textareaResizeObserver.observe(textarea);
+		}
+		this.formView?.element?.addEventListener('keydown', event => {
```

Comment:
Add a comment to mention that it's to prevent automatic dismiss of the form and to allow the user to type multiline.